Tasks/JenkinsQueueJobV2/util.ts

```diff
@@ -40,6 +38,18 @@ export function fail(message: string): void {
 export class FailTaskError extends Error {
 }
 
+export class HttpError extends Error {
```

Comment:
Could you add some short js doc?

Reply:
Thanks for the review point! Added class description.